lib/Job/Async/Worker/Redis.pm

```diff
@@ -252,6 +253,11 @@ sub trigger {
                 })->on_fail(sub {
                     my $failure = shift;
                     $log->errorf("Failed to retrieve job from redis: %s", $failure);
+                    delete $self->{awaiting_job};
+                    delete $self->{redis};
+                    delete $self->{queue_redis};
+                    delete $self->{ryu};
+                    $self->loop->delay_future( after => 1 )->get;
```

Comment:
if the  `get` call is necessary here can you leave a comment? 
why not? 

```
$self->loop->delay_future(after => RECONNET_COOLDOWN)->then(sub ......);
```

Reply:
Agreed. I'll change according to your suggested structure.